packages/appium/docs/en/contributing/index.md

```diff
@@ -16,8 +16,97 @@ already.
 
 ## Ways to Contribute
 
-TODO
+The project offers a variety of ways to contribute, e.g.:
+
+- contributing code
+- improving documentation
+- create educational content (blog posts, tutorials, videos, etc.)
+- spread the good word about the project (e.g. via Twitter)
+- create bugs if you discover them while using Appium
+- make feature requests if you are missing something in the project
 
 ## The Appium Development Process
 
-TODO
+To contribute to the Appium code base make sure to check out the Git repository.
+
+!!! info
+
+    If you are VS Code user you can easily check out the project using [Runme](https://runme.dev/api/runme?repository=https%3A%2F%2Fgithub.com%2Fappium%2Fappium.git&fileToOpen=packages%2Fappium%2Fdocs%2Fen%2Fcontributing%2Findex.md).
+
+It is advised to [fork](https://github.com/appium/appium/fork) before cloning it to your system.
+
+```sh { name=clone }
+export GITHUB_USERNAME=<your-username>
+git clone git@github.com:$GITHUB_USERNAME/appium.git
+```
+
+After cloning you can go ahead and install the project dependencies:
+
+```sh { name=install }
+npm install
+```
+
+From here on there are several things you can do.
+
+### Watch Files
+
+When developing Appium code we have to watch all JavaScript and TypeScript files to re-compile them after every change. You can run this watch process via:
+
+```sh { name=watch, background=true }
+npm run dev
+```
+
+### Start Appium in Dev-Mode
+
+To test your changes you can run Appium in dev mode via:
+
+```sh { name=start }
+npm start
+```
+
+### Run Tests
+
+The project maintains a set of different test variations you can run to verify the quality of the code.
```

Comment:
can we add a note about running tests for a specific package? e.g. `cd packages/$package` and `npm run test`

Reply:
add example to trigger test from a different workspace:
````
You can also run tests for specific workspaces, e.g.:

```sh { name=test-workspace }
export APPIUM_WORKSPACE=@appium/doctor
npm run test:unit -w $APPIUM_WORKSPACE
```
````